modules/rnafusion.py

```diff
@@ -7,6 +7,7 @@
 
 from modules.common import nf_config
 from modules.nextflow import nextflow
+from modules.nextflow import plot_fusion_only_arriba
```

Comment:
`plot_fusion_only_arriba` should be defined in this file, not imported from `modules.nextflow`.